Index: src/main/java/etu2034/framework/Mapping.java
===================================================================
diff --git a/src/main/java/etu2034/framework/Mapping.java b/src/main/java/etu2034/framework/Mapping.java
new file mode 100644
--- /dev/null	
+++ b/src/main/java/etu2034/framework/Mapping.java	
@@ -0,0 +1,30 @@
+package etu2034.framework;
+
+public class Mapping {
+    String className;
+    String method;
+
+    public Mapping(String className, String method) {
+        this.setClassName(className);
+        this.setMethod(method);
+    }
+
+    public Mapping() {
+    }
+
+    public String getClassName() {
+        return className;
+    }
+
+    public void setClassName(String className) {
+        this.className = className;
+    }
+
+    public String getMethod() {
+        return method;
+    }
+
+    public void setMethod(String method) {
+        this.method = method;
+    }
+}
